### PR TITLE
Users/alsehr/module markdown script name

### PR DESCRIPTION
**Pull request by MrMCake**
# Change

Update module table logic + Updated readme

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

**Review by matebarabas — APPROVED** [review]
Approved for initial testing.

File: arm/Microsoft.Network/ipGroups/readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# KeyVault
+# IP Groups
```
Comment (segraef) [inline arm/Microsoft.Network/ipGroups/readme.md:1]: Thanks, noticed the wrong naming as well the last time :)

File: README.md [hunk 2]
```diff
@@ -23,6 +23,113 @@ This repository includes a collection of advanced and curated Components / Units
 * For information on contributing, see [Contribution](<https://github.com/Azure/Modules/wiki#contributing>)
 * File an issue via [GitHub Issues](https://github.com/azure/Modules/issues/new/choose)
 
+## Available modules
```
Comment (segraef) [inline README.md:26]: 🚀

File: arm/README.md [hunk 3]
```diff
@@ -3,100 +3,100 @@ In this section you can find useful information regarding the Modules that are c
 # Available Modules
```
Comment (segraef) [inline arm/README.md:3]: 🚀

**Comment by segraef** [comment]
ignoring linter false-positive

**Review by segraef — APPROVED** [review]
ignoring linter false-positive

**Pull request merged**